code-challenges.test.js

```diff
@@ -27,6 +27,17 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("divisible", () => {
+  it("returns a string stating whether or not an input is divisible by 3", () => {
+
+    expect(divisible(object1.number)).toEqual(`${object1.number} is divisible by three`)
```

Comment:
Remember that the prompt tells you that your function is taking an object as an argument, not a value in that object. Also, in your toEqual, it is best to put exactly what the output of each variable should be, for precise testing.

Reply:
Understood. I should have worked with the value inside the function instead of trying to hard-code it all the way through. Thank you!